mobly/base_test.py

```diff
@@ -126,15 +126,15 @@ def _setup_class(self):
         """Proxy function to guarantee the base implementation of setup_class
         is called.
         """
-        return self.setup_class()
+        self.setup_class()
 
     def setup_class(self):
         """Setup function that will be called before executing any test case in
         the test class.
 
-        To signal setup failure, return False or raise an exception. If
-        exceptions were raised, the stack trace would appear in log, but the
-        exceptions would not propagate to upper levels.
+        To signal setup failure, user asserts or raise your own exception. The
+        stack trace would appear in log, but the exceptions would not propagate
```

Comment:
Ah that's a relic.
I think back in the days tests didn't fail when setup failed and that bug was there for a long time............. Removing it from doc strings